MinecraftClient/McClient.cs

```diff
@@ -202,7 +202,7 @@ private void StartClient(string user, string uuid, string sessionID, string serv
                 client = ProxyHandler.newTcpClient(host, port);
                 client.ReceiveBufferSize = 1024 * 1024;
                 handler = Protocol.ProtocolHandler.GetProtocolHandler(client, protocolversion, forgeInfo, this);
-                Console.WriteLine("Version is supported.\nLogging in...");
+                Translations.WriteLine("mcc.version_supported");

```

Comment:
I think each class should have its own key when possible while of course allowing subkeys, e.g. `mcclient.version_supported`, `mcclient.single_cmd`, `mcclient.error.login_failed` and so on.
This way, we're sure messages will not clash with other classes.

However having some common namespaces such as `error` can be nice too so do what you think is most relevant 🤔

Reply:
I am not sure about that. My thought is information messages from the core part (i.e. login, protocol) will go into `mcc` namespace, error will go into `error`. Commands and ChatBot will be something like `cmd.respawn.msg` and `bot.autoattack.msg`.